src/utils/resin.ts

```diff
@@ -0,0 +1,28 @@
+const RESIN_PREFIX = 'data-resin-';
+
+export function getResinAttribute(name: string): string {
```

Comment:
Having this separate seems like a little bit of overkill. Can we just inline the prefix in the `applyResinTags` function?